app/javascript/local/dropzone-cfg.js

```diff
@@ -12,18 +12,74 @@ DropzoneCfg.prototype.init = function () {
 
   if (!this.$dropzoneContainer || !this.$feedbackContainer) { return }
 
+  // Display of choose files button is conditional on whether JS is enabled
+  document.getElementById('choose_files_button').classList.remove("govuk-visually-hidden")
+
   const self = this
   this.$dropzone = new Dropzone(this.$dropzoneContainer, {
-    paramName: "documents",
-    maxFilesize: 20 // MB
+    paramName: "document",
+    dictDefaultMessage: 'Drag and drop files here or', // TODO: i18n
+    clickable: '#choose_files_button'
   })
 
-  // Basic example of event handling
-  this.$dropzone.on("success", file => {
-    let div = document.createElement("div")
-    div.append(file.name)
-    self.$feedbackContainer.querySelector('.govuk-summary-list').append(div);
+  this.$dropzoneContainerText = document.querySelector('div.dz-message')
+  this.$chooseFilesButton = document.querySelector('button#choose_files_button')
+  this.$dropzoneContainerText.appendChild(this.$chooseFilesButton)
+
+  this.$dropzone.on('addedfile', (file, response) => {
+    let row = createTableRow(file);
+    self.$feedbackContainer.querySelector('.govuk-table__body').append(row);
+  });
+
+  this.$dropzone.on('success', (file, response) => {
+    this.$statusTag = document.getElementById(file.upload.uuid).querySelector(".govuk-tag")
+    this.$statusTag.classList.remove("govuk-tag--yellow")
+    this.$statusTag.classList.add("govuk-tag--green")
+    this.$statusTag.innerHTML = "Uploaded"
   });
+
+  this.$dropzone.on('error', (file, response) => {
+    let error = createErrorMessage(response.error_message)
+    this.$tableCell = document.getElementById(file.upload.uuid)
+    this.$tableCell.append(error)
+    this.$statusTag = this.$tableCell.querySelector(".govuk-tag")
+    this.$tableCell.removeChild(this.$statusTag)
+  })
 }
+function createTableRow(file) {
+  let row = document.createElement("tr")
+  let cell = document.createElement("td")
+  let statusTag = createStatusTag("Uploading")
```

Comment:
Yeah, this is hardcoded here, so I don't think you need it in the translation file